tests/feedback/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from users.models import CustomUser
```

Comment:
Might be worth using the `get_user_model()` helper to save referring to the User model directly. https://docs.djangoproject.com/en/5.1/topics/auth/customizing/#django.contrib.auth.get_user_model
```
from django.contrib.auth import get_user_model

user = get_user_model()
user.objects.create(email="reporter@justice.gov.uk")
```

Reply:
that's a lot nicer, thanks. I'll refactor this once I've got the thing working